tests/test_read_binary.py

```diff
@@ -49,3 +49,9 @@ def test_random_access(shared_datadir):
     data = tsdf.load_binary_from_metadata(metadata[name + ".bin"], 2, 6)
     assert(data.shape == (4, 3))
     assert(data.dtype == "int16")
+
+def test_load_binary_to_dataframe(shared_datadir):
```

Comment:
We should add a test with [hierarchical_meta.json](https://github.com/biomarkersParkinson/tsdf/blob/main/tests/data/hierarchical_meta.json). It's an example where pairs of binaries are used to describe the same type of data over different hours. I am not sure whether the library currently supports that format.